packages/web/src/components/spa/HackAUBG/SponsorsSection/sponsors_section.css

```diff
@@ -130,6 +162,53 @@
     border-color: rgb(163, 104, 37);
 }
 
+@media only screen and (max-width: 900px) {
+    .sponsors-main {
+        margin-bottom: 5%;
+    }
+
+    .sponsors-box-header-container-platinum {
+        padding-top: 2%;
+        min-width: 34vw;
+    }
+
+    .sponsors-box-header-container-gold {
+        padding-top: 2%;
+        min-width: 34vw;
+    }
+
+    .sponsors-box-header-container-custom {
+        padding-top: 2%;
+        min-width: 34vw;
+    }
+
+    .sponsors-box-header-container-silver {
+        padding-top: 2%;
+        min-width: 34vw;
+    }
+
+    .sponsors-box-header-container-bronze {
+        padding-top: 2%;
+        min-width: 34vw;
+    }
```

Comment:
Same here. You apply the same values everywhere, so you can just group the classes.